### PR TITLE
Add additional version adapters

### PR DESCRIPTION
**Pull request by pranav-prakash**
This adds in a few missing adapters that I ran into when converting an LSTM based model (Bidaf) from opset 9 to opset 11. In particular, the following changes were made
* Adding adapters for several compatible ops from 10 -> 11
* Adding a converter for slice op from 9 -> 10, which moves the attributes as inputs

### CODE REVIEW

**Comment by askhade** [comment]
Please add tests for this too.

**Comment by pranav-prakash** [comment]
Added a test for the slice operator adapter. 

I assume that no tests are needed for those ops which used the `CompatibleAdapter`?

File: onnx/version_converter/adapters/slice_9_10.h [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+class Slice_9_10 final : public Adapter {
+  public:
+    explicit Slice_9_10()
+      : Adapter("Slice", OpSetID(9), OpSetID(10)) {}
+
+    void attrToInput(
+        std::shared_ptr<Graph> graph,
+        Node* node,
+        std::vector<int64_t> attr) const {
```
Comment (askhade) [inline onnx/version_converter/adapters/slice_9_10.h:15]: take a reference?

Comment (pranav-prakash) [reply onnx/version_converter/adapters/slice_9_10.h]: Fixed.

**Comment by askhade** [comment]
Please sign the DCO

**Comment by pranav-prakash** [comment]
Done.

**Review by askhade — APPROVED** [review]

**Pull request merged**